### PR TITLE
update documentation for RNA-seq

### PR DESCRIPTION
**Pull request by rhpvorderman**

### CODE REVIEW

File: inputs.md [hunk 1]
```diff
@@ -37,6 +37,10 @@ Please do not use the following inputs:
```
Comment (DavyCats) [inline inputs.md]: I would remove this file. Most of the things mentioned here are pretty general stuff, so they were moved to the central biowdl page.

File: index.md [hunk 2]
```diff
@@ -18,48 +18,38 @@ run `pipeline.wdl` using
 java -jar cromwell-<version>.jar run -i inputs.json pipeline.wdl
 ```
 
-The inputs JSON can be generated using WOMtools as described in the [WOMtools
-documentation](http://cromwell.readthedocs.io/en/stable/WOMtool/). Note that
-not some inputs should not be used! See [this page](inputs.md) for more
-information.
+The inputs.json possible inputs are listed below.
```
Comment (DavyCats) [inline index.md]:
```suggestion
The minimally required inputs are listed below. 
```

File: index.md [hunk 3]
```diff
@@ -18,48 +18,38 @@ run `pipeline.wdl` using
 java -jar cromwell-<version>.jar run -i inputs.json pipeline.wdl
 ```
 
-The inputs JSON can be generated using WOMtools as described in the [WOMtools
-documentation](http://cromwell.readthedocs.io/en/stable/WOMtool/). Note that
-not some inputs should not be used! See [this page](inputs.md) for more
-information.
+The inputs.json possible inputs are listed below. 
+Additional inputs are available. 
+These can be generated using WOMtools as described in the
```
Comment (DavyCats) [inline index.md]:
```suggestion
These can be generated using WOMtool as described in the 
```

File: index.md [hunk 4]
```diff
@@ -18,48 +18,38 @@ run `pipeline.wdl` using
...
 ```
 
-The inputs JSON can be generated using WOMtools as described in the [WOMtools
-documentation](http://cromwell.readthedocs.io/en/stable/WOMtool/). Note that
-not some inputs should not be used! See [this page](inputs.md) for more
-information.
+The inputs.json possible inputs are listed below. 
+Additional inputs are available. 
+These can be generated using WOMtools as described in the 
+[WOMtools-documentation](http://cromwell.readthedocs.io/en/stable/WOMtool/).
```
Comment (DavyCats) [inline index.md]:
```suggestion
[WOMtool-documentation](http://cromwell.readthedocs.io/en/stable/WOMtool/).
```

File: index.md [hunk 5]
```diff
@@ -18,48 +18,38 @@ run `pipeline.wdl` using
...
-| sample.library.preprocessing.<br />baseRecalibrator.<br />knownIndelsSitesVCFs | `Array[File]?` | A VCF file containing known indels. |
-| JointGenotyping.<br />mergeGvcfFiles | `Boolean?` | Whether or not to produce a multisample gVCF file. |
+| sampleConfigFile | `File` | The sample configuration file. See below for more details. |
+| starIndexDir | `String` | The STAR index. |
+| reference | `Dict` | A dictionary containing the following fields <br>"fasta": a fasta file<br>"fai": The fasta index file<br>"dict": the picard index file |
+| refflatFile | `File` | Reference annotation Refflat file. This will be used for expression quantification. |
+| gtfFile | `File` | Reference annotation GTF file. This will be used for expression quantification. |
+| dbsnp| `Dict` | Reference dbSNP VCF file. A dictionary containing the following fields: <br>"file": the dbsnp vcf file  <br>"index": the vcf index file  <br>"md5": Optional(not needed for operation) a md5 file for the vcf file.|
+| dbstrandedness | `String` | Indicates the strandedness of the input data. This should be one of the following: FR (Forward, Reverse),RF (Reverse, Forward) or None: (Unstranded).
```
Comment (DavyCats) [inline index.md]:
```suggestion
| strandedness | `String` | Indicates the strandedness of the input data. This should be one of the following: FR (Forward, Reverse),RF (Reverse, Forward) or None: (Unstranded).
```

File: index.md [hunk 6]
```diff
@@ -105,6 +95,7 @@ measures.
       - **&lt;readgroup>**: Contains QC metrics and preprocessed FastQ files,
       in case preprocessing was necessary.
 - **multisample.vcf.gz**: A multisample VCF file with the variantcalling
+- **multiqc**: Contains the multiqc report.
```
[the remark this answers is not in the document]
Comment (rhpvorderman) [reply index.md]: What do you mean exactly? Searching for `""` does not yield any results

File: index.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (rhpvorderman) [reply index.md]: Done

Comment (rhpvorderman) [reply index.md]: Done

File: inputs.md (hunk 1 above)
Comment (rhpvorderman) [reply inputs.md]: Done

File: index.md (hunk 2 above)
Comment (rhpvorderman) [reply index.md]: Done

File: index.md (hunk 3 above)
Comment (rhpvorderman) [reply index.md]: Done

File: index.md (hunk 4 above)
Comment (rhpvorderman) [reply index.md]: Done

File: index.md [hunk 7]
```diff
@@ -18,48 +18,45 @@ run `pipeline.wdl` using
 java -jar cromwell-<version>.jar run -i inputs.json pipeline.wdl
 ```
 
-The inputs JSON can be generated using WOMtools as described in the [WOMtools
```
Comment (DavyCats) [inline index.md]: I would add an `Input` header here, as a subsection of usage.

File: index.md [hunk 8]
```diff
@@ -18,48 +18,45 @@ run `pipeline.wdl` using
...
+    "index": "The path to the index (.tbi) file associated with the dbSNP VCF"
+  },
+  "pipeline.outputDir": "The path to the output directory",
+  "pipeline.refflatFile": "Reference annotation Refflat file. This will be used for expression quantification.",
+  "pipeline.gtfFile": "Reference annotation GTF file. This will be used for expression quantification.",
+  "pipeline.strandedness": "Indicates the strandedness of the input data. This should be one of the following: FR (Forward, Reverse),RF (Reverse, Forward) or None: (Unstranded)"
+}
+```
 
 ### Sample configuration
```
Comment (DavyCats) [inline index.md]: This should be a subsection of the input section

File: index.md [hunk 9]
```diff
@@ -70,6 +67,61 @@ omitted and R2 values may be omitted in the case of single-end data.
 Multiple readgroups can be added per library and multiple libraries may be
 given per sample.
 
+### Example
```
Comment (DavyCats) [inline index.md]: This should be a subsection of the input section

File: index.md [hunk 10]
```diff
@@ -70,6 +67,61 @@ omitted and R2 values may be omitted in the case of single-end data.
...
+              R2_md5: /home/user/data/patient2/lane1_R2.fq.gz.md5
+          - id: lane2:
+            reads:
+              R1: /home/user/data/patient2/lane2_R1.fq.gz
+              R1_md5: /home/user/data/patient2/lane2_R1.fq.gz.md5
+              R2: /home/user/data/patient2/lane2_R2.fq.gz
+              R2_md5: /home/user/data/patient2/lane2_R2.fq.gz.md5
+```
+
 ## Tool versions
```
Comment (DavyCats) [inline index.md]:
```suggestion
### Dependency requirements and tool versions
```

File: index.md [hunk 11]
```diff
@@ -105,6 +157,7 @@ measures.
       - **&lt;readgroup>**: Contains QC metrics and preprocessed FastQ files,
       in case preprocessing was necessary.
 - **multisample.vcf.gz**: A multisample VCF file with the variantcalling
+- **multiqc**: Contains the multiqc report.
 results.
 
 ## About
```
Comment (DavyCats) [inline index.md]: This header should be removed and the contents of this section should be moved to above usage.

**Pull request merged**